consensus/config/src/committee.rs

```diff
@@ -179,7 +179,12 @@ impl AuthorityIndex {
 
 impl Display for AuthorityIndex {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        f.write_str(self.0.to_string().as_str())
+        if self.value() < 26 {
+            let c = (b'A' + self.value() as u8) as char;
+            f.write_str(c.to_string().as_str())
+        } else {
```

Comment:
This is neat but a bit inconsistent. Also the common case is having > 100 validators. In which context the new format helps?

Reply:
This format makes reading and reasoning through the tests alot easier. So just to compare the two. 

with this change, logs will clearly show the slot as Authority + Round where authority can be reasoned as letter for most of our tests to distinguish it from the round.

```
INFO try_direct_commit: consensus_core::base_committer::base_committer_tests: Try direct commit for leader C6
TRACE try_direct_commit:try_direct_decide{leader=C6}: consensus_core::base_committer: [Committer-L0-R0] A7(OWRg) is not a blame for leader C6
```

and if there are more than 26 validators (which may rarely happen in tests), it will fall back to using a number for the authority. Which makes it okay for prod logging.

```
INFO try_direct_commit: consensus_core::base_committer::base_committer_tests: Try direct commit for leader G6
...
consensus_core::base_committer: [Committer-L0-R0] Z7(vDmc) is not a blame for leader G6
TRACE try_direct_commit:try_direct_decide{leader=G6}: consensus_core::base_committer: [Committer-L0-R0] [26]7(CDzw) is not a blame for leader G6
TRACE try_direct_commit:try_direct_decide{leader=G6}: consensus_core::base_committer: [Committer-L0-R0] [27]7(IHzT) is not a blame for leader G6
TRACE try_direct_commit:try_direct_decide{leader=G6}: 
```

This is compared to the old formatting

```
INFO try_direct_commit: consensus_core::base_committer::base_committer_tests: Try direct commit for leader 26
TRACE try_direct_commit:try_direct_decide{leader=26}: consensus_core::base_committer: [Committer-L0-R0] 7(0;OWRg) is not a blame for leader 26
```

Reply:
INFO try_direct_commit: consensus_core::base_committer::base_committer_tests: Try direct commit for leader G6
...
consensus_core::base_committer: [Committer-L0-R0] Z7(vDmc) is not a blame for leader G6
TRACE try_direct_commit:try_direct_decide{leader=G6}: consensus_core::base_committer: [Committer-L0-R0] [26]7(CDzw) is not a blame for leader G6
TRACE try_direct_commit:try_direct_decide{leader=G6}: consensus_core::base_committer: [Committer-L0-R0] [27]7(IHzT) is not a blame for leader G6
TRACE try_direct_commit:try_direct_decide{leader=G6}

Reply:
INFO try_direct_commit: consensus_core::base_committer::base_committer_tests: Try direct commit for leader G6
...
consensus_core::base_committer: [Committer-L0-R0] Z7(vDmc) is not a blame for leader G6
TRACE try_direct_commit:try_direct_decide{leader=G6}: consensus_core::base_committer: [Committer-L0-R0] [26]7(CDzw) is not a blame for leader G6
TRACE try_direct_commit:try_direct_decide{leader=G6}: consensus_core::base_committer: [Committer-L0-R0] [27]7(IHzT) is not a blame for leader G6
TRACE try_direct_commit:try_direct_decide{leader=G6}: